Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/UserController.cs

```diff
@@ -315,17 +317,17 @@ public IActionResult GetRole()
         /// <param name="email"></param>
         /// <returns></returns>
         [HttpGet]
-        [Route("findbyemail/{email}")]
+        [Route("checkEmailIsExists/{email}")]
         [Authorize(Roles = "Admin")]
-        public async Task<IActionResult> FindByEmail(string email)
+        public async Task<IActionResult> CheckEmailIsExists(string email)
         {
             try
             {
-                return Ok(await _userRepository.FindByEmail(email));
+                return Ok(await _userRepository.CheckEmailIsExists(email + StringConstant.DomainAddress));
             }
             catch (Exception ex)
             {
-                throw ex;
+                return Ok(ex);
```

Comment:
Why are you returning OK when there is an exception? You should never send exception details to client side.